zio-schema/shared/src/test/scala/zio/schema/SchemaGen.scala

```diff
@@ -286,9 +296,9 @@ object SchemaGen {
 
   val anyTransformAndValue: Gen[Random with Sized, TransformAndValue[_]] =
     Gen.oneOf[Random with Sized, TransformAndValue[_]](
-      anySequenceTransformAndValue,
-      anyRecordTransformAndValue,
-      anyEnumerationTransformAndValue
+      anySequenceTransformAndValue
+      // anyRecordTransformAndValue,
```

Comment:
Dead code or intentional?

Reply:
This was intentional. These cause the tests not to halt because of 
```scala
  // TODO: How do we generate a value of a type that we know nothing about?
  val anyEnumerationTransformAndGen: Gen[Random with Sized, EnumerationTransformAndGen[_]] =
    Gen.empty
```

Still need to sort out how to generate arbitraty `Schema.Tranform`s

Reply:
Also, I didn't include in this PR but I think we can actually go back and clean up a lot of the test code now since we should be able to generate arbitrary values really easily and a lot of the existing generators are redundant. 

There is also still the issue of nested `Schema.Optional`. Right now I've kind of hacked around it by not generating `Schema`s that are more the doubly nested and by manually flattening `Option[Option[_]]` when doing equality checks in the tests. But that won't work if you have, for instance, `Right(Some(None))`